### DISCUSSION TITLE
Proper way of upgrading nvm from v1.1.7

### DISCUSSION THREAD

**Discussion by Infinitay** [coreybutler/nvm-windows#953] [Support] [answered]
I'm currently on v1.1.7 of nvm and I wanted to double check if I need to follow the _old_ upgrade guide in which I need to remove all existing node.js versions, and then run the updated installer? If so, is there an easy way I can export any global packages for all my installed node versions thru nvm?

**Comment by coreybutler** [comment, marked as answer]
No, you do not need to remove old Node.js versions. That is only needed when you're installing NVM4W the first time (assuming a prior "regular" version of Node was installed).

You should be able to simply install the latest version over the old version. You can use the main installer or the upgrade installer. Personally, when I'm testing new versions, I just re-run the main installer.

**Reply by Infinitay** [reply]
I didn't realize there was an update executable. Sorry. I ran that and everything worked just fine. Thank you!
